src/advisories/helpers.rs

```diff
@@ -8,7 +8,9 @@ use url::Url;
 /// Whether the database will be fetched or not
 #[derive(Copy, Clone)]
 pub enum Fetch {
-    Allow,
+    /// If this is true, then cargo deny will use the git executable to fetch advisory database.
+    /// If this is false, then it uses a built-in git library.
+    Allow(bool),
```

Comment:
I'd prefer if this was split into distinct variants, the reason the `Fetch` is an enum is because I avoid `bool` since it loses too much information, as shown by the need to comment what the `bool` means in this case.